addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -220,6 +222,14 @@ function factory(dependencies) {
             }
         }
 
+        /**
+         * @private
+         * @param {Object} payload
+         */
+        _handleNotificationMessageUpdate(payload) {
```

Comment:
I'm starting to think for single line insert handler the separate method might be useless